ROS_API.md

```diff
@@ -47,6 +47,7 @@ Below is information about the physical robot API. For the simulation, topics an
 | 🤖🖥️  | `drive_controller`        | Manages mobile robots with a differential or mecanum drive depending on the configuration. It converts velocity commands for the robot body into wheel commands for the base. It also calculates odometry from hardware feedback and shares it. <br/> *[diff_drive_controller/diff_drive_controller](https://github.com/ros-controls/ros2_controllers/tree/master/diff_drive_controller) or [mecanum_drive_controller/mecanum_drive_controller](https://github.com/husarion/husarion_controllers/tree/main/mecanum_drive_controller)* |
 | 🤖🖥️  | `ekf_filter`              | The Extended Kalman Filter node is designed to fuse odometry data from various sources, including wheel encoders, IMU, and GPS. <br/> *[robot_localization/ekf_filter](https://github.com/cra-ros-pkg/robot_localization)*                                                                                                                                                                                                                                                                                                            |
 | 🤖   | `hardware_controller`     | Plugin responsible for communicating with engine controllers via the CAN bus and providing E-Stop functionalities. <br/> *[panther_hardware_interfaces/PantherSystem](./panther_hardware_interfaces/src/panther_system/)*                                                                                                                                                                                                                                                                                                             |
+| 🤖   | `gps`                     | Node responsible parsing NMEA strings and publishing standard ROS NavSat message types. <br/> *[nmea_navsat_driver/nmea_navsat_driver](https://github.com/ros-drivers/nmea_navsat_driver/tree/ros2/src/libnmea_navsat_driver)*                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
We should stick to the convention and change the name of this node to sth like `gps_driver` `nmea_gps_driver` etc...

Reply:
I purposed changes in following commit